wgpu-native/src/command/render.rs

```diff
@@ -338,8 +339,10 @@ pub extern "C" fn wgpu_render_pass_set_vertex_buffers(
             .unwrap();
         vbs.total_size = buffer.size - offset;
     }
-    for vbs in pass.vertex_state.inputs[length..].iter_mut() {
```

Comment:
we should be using `[start_slot as usize + length ..]` here

Reply:
actually, now that we are changing this function to specify a subset of vertex buffers, I think this whole block needs to be removed (sorry for not noticing earlier!)

Reply:
No prob - removed it.